src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -79,4 +101,29 @@ public FindCommand parse(String args) throws ParseException {
         return Optional.of(ParserUtil.parseGenres(genreSet));
     }
 
+    /**
+     * Parses {@code Collection<String> flags} into a {@code Set<Flag>} if {@code flags} is non-empty.
+     * Automatically converts lowercase {@code flagNames} to UPPERCASE
+     * If {@code flags} contain only one element which is an empty string, it will be parsed into a
+     * {@code Set<Flag>} containing zero flags.
+     */
+    private Optional<Flag> parseLoanStateForFind(Collection<String> flags) throws ParseException {
+        requireNonNull(flags);
+
+        if (flags.isEmpty()) {
+            return Optional.empty();
+        }
+        Collection<Flag> flagSet = flags.size() == 1 && flags.contains("") ? Collections.emptySet()
```

Comment:
Lets give the number 1 a meaning here by declaring it a fina constant like 'MAX_FLAG_SIZE'

Reply:
This is code for argument formatting. It's needed because argumentMultimap may contain a "" flag. Doesn't really have a meaning besides that.